cmd/create/cluster/cmd.go

```diff
@@ -490,6 +522,152 @@ func init() {
 		"Enable autoscaling of compute nodes.",
 	)
 
+	// Cluster Autoscaler flags
+	flags.BoolVar(
+		&args.balanceSimilarNodeGroups,
+		"balance-similar-node-groups",
+		false,
+		"Identify node groups with the same instance type and label set, "+
+			"and aim to balance respective sizes of those node groups.",
+	)
+
+	flags.BoolVar(
+		&args.skipNodesWithLocalStorage,
+		"skip-nodes-with-local-storage",
+		true,
+		"If true cluster autoscaler will never delete nodes with pods with local storage, e.g. EmptyDir or HostPat.",
+	)
+
+	flags.IntVar(
+		&args.logVerbosity,
+		"log-verbosity",
+		1,
+		"Autoscaler log level.",
+	)
+
+	flags.IntVar(
+		&args.maxPodGracePeriod,
+		"max-pod-grace-period",
+		0,
+		"Gives pods graceful termination time before scaling down, measured in seconds.",
+	)
+
+	flags.IntVar(
+		&args.podPriorityThreshold,
+		"pod-priority-threshold",
+		0,
+		"The priority that a pod must exceed to cause the cluster autoscaler to deploy additional nodes. "+
+			"Expects an integer, can be negative.",
+	)
+
+	flags.BoolVar(
+		&args.ignoreDaemonsetsUtilization,
+		"ignore-daemonsets-utilization",
+		false,
+		"Should cluster-autoscaler ignore DaemonSet pods when calculating resource utilization for scaling down.",
+	)
+
+	flags.StringVar(
+		&args.maxNodeProvisionTime,
+		"max-node-provision-time",
+		"",
+		"Maximum time cluster-autoscaler waits for node to be provisioned. "+
+			"Expects string comprised of an integer and time unit (ns|us|µs|ms|s|m|h), examples: 20m, 1h.",
+	)
+
+	// flags.StringVar(
+	flags.StringSliceVar(
+		&args.balancingIgnoredLabels,
+		"balancing-ignored-labels",
+		nil,
+		"A comma-separated list of label keys that cluster autoscaler should ignore when considering node group similarity.",
+	)
+
+	// Resource Limits
+	flags.IntVar(
+		&args.resourceLimits.MaxNodesTotal,
+		"max-nodes-total",
+		1000,
+		"Maximum amount of nodes in the cluster (not only autoscaled ones).",
+	)
+
+	flags.IntVar(
+		&args.resourceLimits.Cores.Min,
+		"min-cores",
+		0,
+		"Minimum number of cores to deploy in the cluster.",
+	)
+
+	flags.IntVar(
+		&args.resourceLimits.Cores.Max,
+		"max-cores",
+		100,
+		"Maximum number of cores to deploy in the cluster.",
+	)
+
+	flags.IntVar(
+		&args.resourceLimits.Memory.Min,
+		"min-memory",
+		0,
+		"Minimum amount of memory, in GiB, in the cluster.",
+	)
+
+	flags.IntVar(
+		&args.resourceLimits.Memory.Max,
+		"max-memory",
+		4096,
+		"Maximum amount of memory, in GiB, in the cluster.",
+	)
+
+	// TODO: handle GPU limitations
+
+	// Scale down Configuration
+
+	flags.BoolVar(
+		&args.scaleDown.Enabled,
+		"scale-down-enabled",
+		true,
+		"Should Cluster Autoscaler scale down the Cluster.",
+	)
+
+	flags.StringVar(
+		&args.scaleDown.UnneededTime,
+		"scale-down-unneeded-time",
+		"",
+		"How long a node should be unneeded before it is eligible for scale down.",
+	)
+
+	flags.Float64Var(
+		&args.scaleDown.UtilizationThreshold,
+		"scale-down-utilization-threshold",
```

Comment:
Please add these as constants instead to make sure the consts are used when setting up the manual command for creating the cluster with the same config, and everywhere else where it references these values

Reply:
Not sure If I am understanding this correctly. These are CLI flags passed by the user and therefore variable. Could you expand on how I would set them as constants?

Reply:
Done!